### PR TITLE
#63: Instructions Module

### PR DESCRIPTION
**Pull request by NathHorrigan**
This PR implements the Instructions Module and all of it's associated screens.

Ticket: https://projects.torchbox.com/projects/flare-rebuild/tickets/63

### CODE REVIEW

File: src/containers/InstructionsContainer.tsx [hunk 1]
```diff
@@ -0,0 +1,158 @@
...
+    onNextInstruction()
+  }
+
+  if (mod.advancedVolumeCalibration) {
+    introScreens = introScreens.concat(() => (
+      <VolumeCalibrationScreen onFinishCalibration={onFinishCalibration} />
+    ))
+  }
+
+  // All slides for trual setup
```
Comment (mixxorz) [inline src/containers/InstructionsContainer.tsx]: Typo: trial

File: src/containers/InstructionsContainer.tsx [hunk 2]
```diff
@@ -0,0 +1,158 @@
...
+    : []
+
+  // Get the correct screen
+  const IntroScreen = introScreens?.[currentAppInstruction]
+  const TrialScreen = trialScreens?.[currentTrialInstruction]
+  const CurrentScreen = IntroScreen ?? TrialScreen
+  const inRatingPhase = IntroScreen === undefined && TrialScreen !== undefined
+
+  // If no screens left then finish module
+  if (IntroScreen === undefined && TrialScreen === undefined) {
```
Comment (mixxorz) [inline src/containers/InstructionsContainer.tsx]: Shouldn't this be in a `useEffect`? Not sure if we should be calling `onModuleComplete` in the render code path directly.

**Review by mixxorz — COMMENTED** [review]
Looking good. Thanks Nathan!

File: src/containers/InstructionsContainer.tsx (hunk 2 above)
Comment (NathHorrigan) [reply src/containers/InstructionsContainer.tsx]: Good Shout!

File: src/containers/InstructionsContainer.tsx (hunk 1 above)
Comment (NathHorrigan) [reply src/containers/InstructionsContainer.tsx]: 👍

**Pull request merged**